webapp/src/Command/ImportEventFeedCommand.php

```diff
@@ -1046,15 +996,18 @@ protected function importProblem(array $event): void
             return;
         }
 
-        $this->logger->info('Importing problem %s event %s', [ $event['op'], $event['id'] ]);
+        $this->logger->info('Validating problem %s event %s', [ $event['op'], $event['id'] ]);
 
         $problemId = $event['data']['id'];
 
         // First, load the problem
         /** @var Problem $problem */
         $problem = $this->em->getRepository(Problem::class)->findOneBy(['externalid' => $problemId]);
         if (!$problem) {
-            $this->logger->error('Problem %s not found, cannot import', [ $problemId ]);
+            $this->logger->warning(
+                'Problem with external ID %s nout found in DOMjudge',
```

Comment:
typo.

Reply:
Is this still in the last commit? If not, let's ignore it 🙈